_episodes/12-cluster.md

```diff
@@ -62,27 +62,14 @@ Go ahead and log in to the cluster: {{ site.remote.name }} at {{ site.remote.loc
 ```
 {: .bash}
 
-Remember to replace `yourUsername` with the username supplied by the instructors. You will be asked
-for your password. But watch out, the characters you type are not displayed on the screen.
+Remember to replace `yourUsername` with the username supplied by the instructors. You may be asked
+for your password. Watch out: the characters you type are not displayed on the screen.
```

Comment:
The characters you type *at* *the* *password* *prompt* are not displayed on the screen.

Reply:
f14c07b